symengine/tests/eval/test_eval_mpfr.cpp

```diff
@@ -77,5 +78,15 @@ TEST_CASE("precision: eval_mpfr", "[eval_mpfr]")
     REQUIRE(mpfr_cmp_d(a, 2.00000000000001) == -1);
     REQUIRE(mpfr_cmp_d(a, 1.99999999999999) == 1);
 
+    r = loggamma(integer(3));
```

Comment:
This is automatically simplified right? So, this is not evaluating `loggamma(3)`, but `log(gamma(3))`